src/pages/proposal-commons/proposal-details/proposal-details.tsx

```diff
@@ -26,6 +26,33 @@ import NotFoundPage from '../../not-found';
 import { handleTransactionError, images, messages, useScrollToTop } from '../../../utils';
 import ExternalLink from '../../../components/external-link';
 
+interface ProposalDescriptionProps {
+  description: string;
+}
+
+const ProposalDescription = (props: ProposalDescriptionProps) => {
+  const { description } = props;
+
+  // The regex is intentionally simpler than usual URL checking regexes.
+  //
+  // The whole regex expression must be quoted, otherwise the delimeter (the URL) will be excluded.
+  const parts = description.split(/(https?:\/\/[^\s]+)/g);
```

Comment:
Do we maybe want to include IPFS links too?

Reply:
I've asked this in https://api3workspace.slack.com/archives/C02ALA11APM/p1629109315030300?thread_ts=1629097143.026700&cid=C02ALA11APM but @bbenligiray was against